frontend/src/metabase/visualizations/echarts/pie/model/types.ts

```diff
@@ -0,0 +1,24 @@
+import type { ColumnDescriptor } from "metabase/visualizations/lib/graph/columns";
+
+export interface PieColumnDescriptors {
+  metricDesc: ColumnDescriptor;
```

Comment:
really nit, and maybe we should go the other way: I did not use the `Desc` suffix because with typescript it is self-descriptive enough and without the suffix it is a bit cleaner. In your opinion, does it improve readability?

Reply:
For me personally it helps a lot. When reading our codebase I find that we have very ambiguous names. If I see just the name `metric`, that could mean a lot of things. Is it referring to the metric column, the name of the metric column (like in viz settings), or the metric descriptor? In our codebase, it can be any one of those things, and across different use cases it will be all of them, which is very confusing to me.

Sure typescript helps, but it requires me to mouseover, which is inconvenient when reading large blocks of code. Ideally I should just be able to look at the code and understand what everything is, without clicking in to everything.

`metricDesc`, unlike just `metric`, immediately removes the ambiguity and tells me what this variable is.